R/dvhydrograph-render.R

```diff
@@ -20,64 +15,65 @@ createDVHydrographPlot <- function(reportObject){
     'startDate',
     'endDate',
     'isInverted',
-    'timezone'
+    'timezone',
+    'stationId',
+    'stationName',
+    'title'
   )
 
+  #Get Necessary Report Metadata
   if(validateFetchedData(metaData, "metadata", requiredMetadataFields)){
-    #Get Necessary Report Metadata
     timezone <- fetchReportMetadataField(reportObject, 'timezone')
     excludeZeroNegativeFlag <- fetchReportMetadataField(reportObject, 'excludeZeroNegative')
     excludeMinMaxFlag <- fetchReportMetadataField(reportObject, 'excludeMinMax')
     invertedFlag <- fetchReportMetadataField(reportObject, 'isInverted')
+    excludeZeroNegativeFlag <- ifelse(is.null(excludeZeroNegativeFlag), FALSE, TRUE)
+    excludeMinMaxFlag <- ifelse(is.null(excludeMinMaxFlag), FALSE, TRUE)
+    invertedFlag <- ifelse(is.null(invertedFlag), FALSE, TRUE)
```

Comment:
These are addressed in my Five Year GW refactor. I added a parseReportMetadataField() function that can be supplied with a default value to return if the field is not found, so now instead of a fetch and then null check it just uses one parse line.